### ISSUE TITLE
Group error cloning site between Portal environments.

### ISSUE THREAD

**Issue by mgress12** [Esri/hub-py#132]
I get the following error when attempting to clone a site between our dev and test Portal environments. Both are at v10.9.1

Exception: Unable to create group.
You do not have permission to create groups with external members.
(Error Code: 400)

The group associated with the dev site only includes myself as a member, an admin. I'm an admin in the test Portal as well with the same username. 

Any workarounds related to this cloning issue?

**Comment by JSchneids** [comment]
I am running into this issue as well when using two brand new 10.9.1 installs.

**Comment by ManushiM** [comment]
@mgress12 @JSchneids can you please send me a screenshot of the executed command and the error you are observing? 
I do not see errors on my end, and would like to see where the clone is failing for you.

**Comment by mgress12** [comment]
@ManushiM - I was able to get around this by editing sites.py, changing the entry for membershipAccess to "org" under the clone function for the collab_group_dict parameters. The original setting was "collaboration" which was throwing things off for me. 

_collab_group_dict = {
                "title": subdomain + ' Core Team', 
                "tags": ["Hub Group", "Hub Initiative Group", "Hub Site Group", "Hub Core Team Group", "Hub Team Group"], 
                "access":"org",
                "capabilities":"updateitemcontrol",
                **"membershipAccess": "**org**",**
                "snippet": "Members of this group can create, edit, and manage the site, pages, and other content related to hub-groups."

Below is the command I used after connecting to both Portals, and getting the site to be cloned.

- site_cloned = testhub.sites.clone(site)
- site_cloned.item

It created groups in the target Portal, the site application, and site pages. Not sure what the expected behavior should be beyond that. 
Looks like I need to clone other content/items from the source Portal and rewire the cloned site to point to those. Also, rename the created groups and associated site application and pages.
